src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+    private final String from;
+    private final String to;
+
+    public Event(String title, String from, String to) {
+        super(title);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + "(from: " + this.from + " to: " + this.to + ")";
```

Comment:
Instead of using string concatenation, it is better to use String.format for clearer and more maintainable code.

```suggestion
        return String.format("[E]%s(from: %s to: %s)", super.toString(), this.from, this.to);
```